.ci/Jenkinsfile

```diff
@@ -43,10 +43,10 @@ pipeline {
             stash allowEmpty: true, name: 'source', useDefaultExcludes: false
           }
         }
-        /**
-        Check the source code.
-        */
-        stage('Check it contains Buildkite pipeline') {
+        stage('Check if the PR contains Buildkite pipeline') {
+          when {
+            changeRequest()
+          }
```

Comment:
This stage is intended to check if the PRs are "old", it means if they are based on a commit previous to the buildkite pipeline.

I think it could be just run in case of PRs (`changeRequest()`). The last run when a tag was pushed failed https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Felastic-package/detail/v0.75.0/1/pipeline/